components/infobox/commons/infobox.lua

```diff
@@ -34,13 +35,7 @@ function Infobox:create(frame, gameName, forceDarkMode)
 end
 
 function Infobox:categories(...)
```

Comment:
Is this function needed? Can the caller invoke ext `add_category` directly?

Reply:
It's something that could be refactored later. It's also possible that the function might be kept and more logic is added to it (#1581)